### PR TITLE
WIP: CODEOWNERS

### PR DESCRIPTION
**Pull request by NotMoni**
Signed-off-by: Moni <usmoni@gmail.com>

**What is the purpose of this pull request?**

- [ ] Documentation update
- [ ] Bug fix
- [x] New feature
- [ ] Other, please explain:

**What changes did you make? (provide an overview)**
Started CODEOWNERS

**Which issue (if any) does this pull request address?**
#852 
**Is there anything you'd like reviewers to know?**
This is a v1 and more revisions are to be made. This is not perfect yet.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# @node-fetch/Core
+* @bitinn @xxczaki @Richienb @gr2m @jimmywarting
+
+# JS Files
+*.js @tinovyatkin @xxczaki @Richienb @jimmywarting @xxczaki
+
+# Docs
+docs/* @NotMoni @xxczaki @Richienb
+
+# Github
```
Comment (yg) [inline .github/CODEOWNERS:10]: Brand :unicorn: 

```suggestion
# GitHub
```

Comment (NotMoni) [reply .github/CODEOWNERS]: @yg Thanks !

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# @node-fetch/Core
+* @bitinn @xxczaki @Richienb @gr2m @jimmywarting
+
+# JS Files
+*.js @tinovyatkin @xxczaki @Richienb @jimmywarting @xxczaki
```
Comment (xxczaki) [inline .github/CODEOWNERS:5]:
```suggestion
*.js @tinovyatkin @xxczaki @Richienb @jimmywarting
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# @node-fetch/Core
+* @bitinn @xxczaki @Richienb @gr2m @jimmywarting
```
Comment (xxczaki) [inline .github/CODEOWNERS:2]: I'm not sure I understand this line - what is its purpose? Can't it be replaced with just `@node-fetch/core`?

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+*.js @tinovyatkin @xxczaki @Richienb @jimmywarting @xxczaki
+
+# Docs
+docs/* @NotMoni @xxczaki @Richienb
+
+# GitHub
+.github/* @NotMoni @tinovyatkin @jimmywarting
+
+# TS Types
+@types/* @tinovyatkin @jimmywarting @xxczaki
```
Comment (xxczaki) [inline .github/CODEOWNERS:14]:
```suggestion
@types/* @tinovyatkin @jimmywarting @xxczaki @Richienb
```

File: .github/CODEOWNERS (hunk 3 above)
Comment (NotMoni) [reply .github/CODEOWNERS:2]: @xxczaki Yep. I just realized that. But im not sure if this file supports teams.

Comment (xxczaki) [reply .github/CODEOWNERS:2]: @NotMoni I think it does, see https://github.com/gatsbyjs/gatsby/blob/master/CODEOWNERS

Comment (NotMoni) [reply .github/CODEOWNERS]: @xxczaki Thanks for that suggestion. It has now been updated.

**Comment by JefferyHus** [comment]
Not yet sure about the tagged people there. So this is just about who can, or have the rights to open PRs and fix bugs..etc?

**Comment by NotMoni** [comment]
@JefferyHus Nope. These are the people the automatically get assigned to review the PR's. Althought other can review too.

**Comment by JefferyHus** [comment]
Makes sense, thanks

**Comment by NotMoni** [comment]
cc/ @Richienb @tinovyatkin @jimmywarting

**Comment by tinovyatkin** [comment]
@NotMoni the PR is in Draft status, so, I was assuming it's not ready for review yet (and as far I see you are ignoring my recommendation to not do broad assignment and assigned the whole team for all `.js` that is big 👎  from my point of view), while @LinusU who did a great contribution for our TypeScript is not there...

**Comment by NotMoni** [comment]
@tinovyatkin I don't know what everyone is good at. I was trying to get some suggestions as to how I should move people around...

@LinusU I'm sorry for not including you. 😢

**Comment by NotMoni** [comment]
Closing this. 🕘

**Pull request closed without merging**